aries-site/src/pages/index.js

```diff
@@ -1,69 +1,135 @@
+        pad={{ horizontal: calcPad(size) }}
+        direction="row"
+      >
+        {size !== 'small' && <Box width="small" />}
+        <Box>
+          <Image src={image} fit="contain" />
+        </Box>
+      </Box>
+      <Box height={{ min: 'medium' }} justify="center">
+        <Grid
+          gap="large"
+          columns={size === 'small' ? ['auto'] : ['3/4', 'auto']}
+        >
+          <Card background="none" elevation="none">
+            {children}
+          </Card>
+          {size !== 'small' && (
+            <Card background="none" elevation="none" height="small" />
+          )}
+        </Grid>
+      </Box>
+    </Stack>
+  );
+};
 
-cards.forEach(card => {
-  if (featured.includes(card.name)) featuredCards.push(card);
-  else cardOrder.push(card);
-});
+Intro.propTypes = {
+  children: PropTypes.node,
+};
 
-featuredCards.sort(
-  (a, b) => featured.indexOf(a.name) - featured.indexOf(b.name),
-);
+const Index = () => {
+  const size = useContext(ResponsiveContext);
+  const darkMode = useDarkMode();
+  const videoRef = useRef(null);
 
-cardOrder.splice(0, 0, ...featuredCards);
+  const [videoEnabled, setVideoEnabled] = useState(false);
 
-const Index = () => (
-  <Layout
-    backgroundImage={{
-      src: { dark: '/cards-dark.svg', light: '/cards.svg' },
-      alt: 'HPE Design System',
-      margin: { top: '50px', left: '-150px' },
-      style: { transform: 'scale(1.4)', transformOrigin: 'top left' },
-      small: {
-        margin: { left: '-75px', top: '-75px' },
-      },
-      useGrid: true,
-    }}
-    title={title}
-    isLanding
-  >
-    <Meta title={title} description={pageDetails.seoDescription} />
-    <Box gap="large">
-      <PageIntro>
-        <Box justify="center" fill>
-          <Heading margin="none">HPE Design System</Heading>
-          <Paragraph size="large" fill>
-            The Design System was created to empower designers, developers, and
-            others to contribute to making great experiences for the customer.
-          </Paragraph>
+  return (
+    <Layout title={title} isLanding pad={{}} width={{}}>
+      <Meta title={title} description={pageDetails.seoDescription} />
+      <Box>
+        <Intro>
+          <Box pad={calcPad(size)} width={{ max: '900px' }}>
+            {size === 'small' && (
+              <Card background="none" elevation="none" height="small" />
+            )}
+            <Heading margin="none" size="large">
+              Design, develop and deliver
+            </Heading>
+            <Paragraph size="xlarge">
+              Created to empower designers, developers, and others to quickly
+              create accessible user experiences
+            </Paragraph>
+          </Box>
+        </Intro>
+        <Stack guidingChild="last">
+          <Box background="background-front" margin={{ top: 'xlarge' }} fill />
+          <Featured />
+        </Stack>
+        <WhatIs />
+        <Box
+          fill
+          onClick={() => {
```

Comment:
Any way we could make it so clicking again could pause the video? With the video taking up the whole screen, it can be difficult to find the controls to pause it

Reply:
Yes, there is a pause() method. The question is how to do it and still have the controls on the bottom while it's playing. I put the controls in once it's playing because it lets you adjust the position in the video, but it would be a little work to make that click not interfere with the position control and other controls like maximize and such. Thoughts?

Reply:
hmm what about showing the controls below instead? `controls={videoEnabled ? 'below' : false}`

Would that still cause interference?

Reply:
Yeah, the click still is basically for the entire video element in the box. I'll play with it but the other detail is figuring out for sure if it's playing or paused since they can pause it via the controls. I'll see what I can do.

Reply:
Mmm got it, tricky. It's mostly apparent on bigger screens. once I shrink my window down, the context of where I'd pause/play is much more visible. Could be a design question as well. See below how it looks when I have it full-width (15" Mac) vs shrinking it down:

https://user-images.githubusercontent.com/12522275/107095411-9f0f6c00-67bd-11eb-9298-858c7e976580.mov

Reply:
Got it working. It was easier than expected since the event.target is actually different for the controls.

Reply:
interesting. I wonder if it would make sense to make the `<video>` max-height be the viewport height?

Reply:
This actually works pretty well

Reply:
Functionality works great! Wondering if the controls should align with the width of the video? See below:
<img width="1670" alt="Screen Shot 2021-02-05 at 3 53 57 PM" src="https://user-images.githubusercontent.com/12522275/107100903-85c0ec80-67ca-11eb-9fb5-580ab2065a69.png">

Reply:
The grey is the extra padding on the sides when the video is restricted to viewport height

Reply:
Yup. That seems to be how the video component scales itself